webpack.config.js

```diff
@@ -1,24 +1,18 @@
 const path = require('path');
 const webpack = require('webpack');
-const WebpackGitHash = require('webpack-git-hash');
 const fs = require('fs');
 const pkg = require('./package.json');
 const TerserPlugin = require('terser-webpack-plugin');
-const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer')
+    .BundleAnalyzerPlugin;
 
 // Import config file
-let isAWS = process.env.NODE_TYPE == 'aws';
-let _config = null;
-if (isAWS) {
-    _config = require('./apache/config.json');
-} else {
-    _config = require(process.env.SCOPE_CONFIG || './config.json');
-}
```

Comment:
This is not reflected in the docs. We should then I guess remove section and clearly state that we (currently) do not support AWS